lib/browser/api/utility-process.ts

```diff
@@ -6,49 +6,46 @@ import { MessagePortMain } from '@electron/internal/browser/message-port-main';
 const { createProcessWrapper } = process._linkedBinding('electron_browser_utility_process');
 
 class IOReadable extends Readable {
-  _shouldPush: boolean = false;
-  _data: (Buffer | null)[] = [];
-  _resume: (() => void) | null = null;
-
-  _storeInternalData (chunk: Buffer | null, resume: (() => void) | null) {
-    this._resume = resume;
-    this._data.push(chunk);
-    this._pushInternalData();
+  #shouldPush: boolean = false;
+  #data: (Buffer | null)[] = [];
+  #resume: (() => void) | null = null;
+
+  storeInternalData (chunk: Buffer | null, resume: (() => void) | null) {
+    this.#resume = resume;
+    this.#data.push(chunk);
+    this.#pushInternalData();
   }
 
-  _pushInternalData () {
-    while (this._shouldPush && this._data.length > 0) {
-      const chunk = this._data.shift();
-      this._shouldPush = this.push(chunk);
+  #pushInternalData () {
+    while (this.#shouldPush && this.#data.length > 0) {
+      const chunk = this.#data.shift();
+      this.#shouldPush = this.push(chunk);
     }
-    if (this._shouldPush && this._resume) {
-      const resume = this._resume;
-      this._resume = null;
+    if (this.#shouldPush && this.#resume) {
+      const resume = this.#resume;
+      this.#resume = null;
       resume();
     }
   }
 
   _read () {
-    this._shouldPush = true;
-    this._pushInternalData();
+    this.#shouldPush = true;
+    this.#pushInternalData();
   }
 }
 
 export default class UtilityProcess extends EventEmitter {
-  _handle: any;
-  _stdout: IOReadable | null | undefined = new IOReadable();
-  _stderr: IOReadable | null | undefined = new IOReadable();
+  #handle: ElectronInternal.UtilityProcessWrapper | null;
+  #stdout: IOReadable | null | undefined = new IOReadable();
+  #stderr: IOReadable | null | undefined = new IOReadable();
```

Comment:
a bit weird that these can be both null and undefined. can we narrow it to just one of those?

Reply:
I based these on https://nodejs.org/dist/latest-v16.x/docs/api/child_process.html#subprocessstdout, I like the idea of being just `IOReadable | null` to be consistent with other properties.

Reply:
https://github.com/electron/electron/pull/34980/commits/6430fc8a68285f9499df08583308c87dd5b0ff0e